include/mockgyver.hrl

```diff
@@ -5,22 +5,53 @@
 
 -compile({parse_transform, mockgyver_xform}).
 
+-ifdef(OTP_RELEASE).
+-define(mockgyver_get_stacktrace__(),
+        try error(get_stack)
+        catch error:get_stack:St -> St
+        end).
+-else. % OTP_RELEASE
+-define(mockgyver_get_stacktrace__(),
+        try error(get_stack)
+        catch error:get_stack -> erlang:get_stacktrace()
+        end).
+-endif. % OTP_RELEASE.
```

Comment:
Similar question about what Erlang version to aim for. For now, it is the same as in #18, but it is easy to change.

Reply:
As in #18 is good. I just merged that one as is.